src/test/java/bio/terra/service/filedata/DrsServiceTest.java

```diff
@@ -1094,6 +1094,63 @@ void testUniqueExtraction() {
         () -> DrsService.extractUniqueDrsObjectValue(drsObjects, DRSObject::getSize));
   }
 
+  @Test
+  void testExtractFirstDrsObjectValue() {
+    DRSObject drsObject1 =
+        createFileDrsObject(
+            "v2_file1",
+            "/my/path/file1.txt",
+            123L,
+            "foomd5",
+            CloudPlatform.GCP,
+            GoogleRegion.ASIA_SOUTH1,
+            Instant.parse("2025-01-01T00:00:00.00Z"));
+    drsObject1.setDescription("description1");
+    DRSObject drsObject2 =
+        createFileDrsObject(
+            "v2_file1",
+            "/my/path/file1.txt",
+            123L,
+            "foomd5",
+            CloudPlatform.GCP,
+            GoogleRegion.ASIA_SOUTH1,
+            Instant.parse("2025-01-01T00:00:00.00Z"));
+    drsObject2.setDescription("description2");
+    List<DRSObject> drsObjects = List.of(drsObject1, drsObject2);
+    assertThat(
+        "value from first object is correctly returned",
+        DrsService.extractFirstDrsObjectValue(drsObjects, DRSObject::getDescription),
```

Comment:
In addition to testing `extractFirstDrsObjectValue`, you should add a test to `testMergeDrsObjects()` since we need to verify that `DrsService.mergeDRSObjects()` works correctly.